pex/resolver.py

```diff
@@ -28,7 +28,7 @@
 from pex.util import CacheHelper
 
 
-class Untranslateable(Exception):
+class Untranslatable(Exception):
```

Comment:
Unclear whether this is a public API... but I don't see any usages of the old name in `pants`, so I'm fine with just renaming.